app/src/main/java/org/jellyfin/mobile/utils/WebViewUtils.kt

```diff
@@ -71,3 +73,19 @@ fun WebSettings.applyDefault() {
     javaScriptEnabled = true
     domStorageEnabled = true
 }
+
+/**
+ * Opens the requested file from the application's assets directory.
+ *
+ * By default Android doesn't provide the javascript MIME type,
+ * return "application/javascript" when required.
+ *
+ * @see WebViewAssetLoader.AssetsPathHandler.handle
+ */
+fun WebViewAssetLoader.AssetsPathHandler.inject(path: String): WebResourceResponse? {
+    return handle(path)?.apply {
+        if (path.endsWith(".js", ignoreCase = true)) {
+            mimeType = "application/javascript"
+        }
+    }
+}
```

Comment:
I'd argue that ignoring case isn't necessary, but I suppose it doesn't hurt either. Still, I suggest collapsing the body to an `=` statement / removing the return.